content/blog/announcing-pulumi-esc-github-action/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Announcing the Pulumi ESC GitHub Action"
```

Comment:
Does the action not have a "migrate GH secrets to ESC" sub action?

Reply:
No, it does not. I can add the bash script I used to build it in the docs. We can leave building that into the action as a future improvement.